frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -123,25 +124,36 @@ function TweakInfoWidget:init()
 
     local buttons = {
         {
-            text = _("Close"),
-            callback = function()
-                UIManager:close(self)
-            end,
+            {
+                text = self.is_tweak_registered and _("Don't show in action list") or _("Show in action list"),
+                callback = function()
+                    self.toggle_tweak_registered_callback()
```

Comment:
The wording "tweak registered" in this pure styletweaks context is a bit confusing - I would mention "dispatcher" in the variables/settings/methods, ie. `is_dispatcher_registered`.

Reply:
It corresponds with `self.registered_tweaks` and `style_tweaks_registered` G_setting.

Reply:
Yes, I saw that. But even `style_tweaks_registered` is a bit anonymous when just viewing the settings. If `style_tweaks_dispatcher_registered` is too long, just seeing "dispatcher" is better than "registered" (into what?). `style_tweaks_dispatcher` ? `style_tweaks_in_dispatcher` ?  so-so :/
`registered` sounds too alike `enabled` when just studying this readerstyletweak.lua module.

Reply:
Feels like `style_tweaks_in_dispatcher`, `is_tweak_in_dispatcher`, `self.tweaks_in_dispatcher` would read well.
If you really feel like sticking with "..._registered", please add some comment where this thing is created/read/saved, `-- registered in dispatcher`, so we don't have to follow it all the way to where it is neighboured by `dispatcherRegisterStyleTweak()` to know it is actually in Dispatcher that it is registered :)

Reply:
I'm not stuck, just have no good idea. Let it be "in_dispatcher".